Added channels, changed orders to keep the same hierarchy the server has, modified the readme's and obsolete channel descriptions, added graven to reférents lists

padmeon opened the pull request:
This commit is nearly a complete overhaul of this repository.
- Added channels : roles, questions-fréquentes, anciennes-vidéos, clash of code, dart | flutter

- Changed the orders of the different channels to keep the same hierarchy since new channels were added, example: 
the channel `annonces` was number two and became number 4 because of the channels `roles` and `questions-fréquentes`

- Added the new channels to the readme's

- Modified some obsolete descriptions of channels (like the `autres` channel that doesn't exist anymore because it is now a forum) Deleted some channels that don't exist/are not visible anymore

- Added Graven in the référents lists (like `applications-mobiles` and `dart | flutter`)

Good luck reviewing everything 😉

padmeon committed:
Lots of modifications made (a near full overhaul of the project)

This commit is nearly a complete overhaul of this repository.
Added channels : roles, questions-fréquentes, anciennes-vidéos, clash of code, dart | flutter
Changed the orders of the different channels to keep the same hierarchy since new channels were added (example the channel annonce was number two and became number 4 because of the channels roles and question) and changed their numbers in their filenames.
Added the new channels to the readme's
Modified some obsolete descriptions of channels (like the `autres` channel that doesn't exist anymore because it is now a forum)
Deleted some channels that don't exist/are not visible anymore

g-clems commented on channels/1 - Système/2_roles.md:
```diff
@@ -0,0 +1,6 @@
+# `#🎭︱roles`
+Ce salon contient un système de 
```
Description incomplète

SkyrixFr replied:
```suggestion
Ce salon contient un système de rôles permettant aux utilisateurs de définir les languages pratiqué par les membres du serveur
```

g-clems commented on channels/4 - Salons d'aide/16_autres.md:
```diff
@@ -1,6 +1,6 @@
-Le salon `#❔︱autres` est un salon vous permettant de poser vos questions sur les 
-langages non-couverts par un salon
+# `#❔︱autre︱other`
+Le salon `#❔︱autre︱other` est un forum vous permettant de poser vos questions, sur les 
```
virgule en trop

g-clems commented on channels/4 - Salons d'aide/16_autres.md:
```diff
@@ -1,6 +1,6 @@
+# `#❔︱autre︱other`
+Le salon `#❔︱autre︱other` est un forum vous permettant de poser vos questions, sur les 
+langages non-couverts par un salon, sous la forme de posts
 
```
Guidelines manquante

padmeon replied:
Qu'est ce qu'une guideline?

padmeon replied:
C'est d'expliquer comment on ouvre un post?

g-clems replied:
Elles sont visible en cliquant sur "nouveau post"

padmeon committed:
Modifications made after being requested by Clems

Finished the description of the ``roles`` channel
Modified the filename of ``autres | other`` channel
Modified descriptions that weren't really french
Modified the README with the list of all the channels (forgot to do it in the other commit)

padmeon commented:
All the changes asked were made and more was done (things I forgot to add in the old commit)

RedsTom committed:
Update 3_questions-fréquentes.md
